api/app/wildfire_one/wfwx_api.py

```diff
@@ -128,7 +128,7 @@ async def get_detailed_stations(time_of_interest: datetime):
                     temperature=daily.get('temperature'),
                     relative_humidity=daily.get('relativeHumidity'))
                 record_type = daily.get('recordType').get('id')
-                if record_type == 'ACTUAL':
+                if record_type in ['ACTUAL', 'MANUAL']:
```

Comment:
What is manual?

Reply:
Manual is where someone goes into WFWX API and manually overrides whatever values were automatically reported by the sensors at a weather station (I think this only ever happens when there's something wrong with a sensor). There were a lot of them recently in WFWX API and it was breaking things